Makefile

```diff
@@ -61,6 +61,17 @@ deps:
 	git submodule update --init --recursive
 	$(KEVM_MAKE) llvm-deps
 
+start-vm:
+	PATH="$$PATH:$(shell pwd)/deps/evm-semantics/.build/defn/vm" node ./deps/ganache-cli/cli.js &
```

Comment:
I think there must be something better than `$(shell pwd)`, I'm open to suggestions.

Reply:
@gtrepta let's do (above in the `Makefile`, where other vars are declared) do:

```
PATH:=$(PATH):$(CURDIR)/deps/evm-semantics/.build/defn/vm
export PATH
```

Then this line can just be `node ./deps/ganache-cli/cli.js &`